crates/summit/static/style.css

```diff
@@ -0,0 +1,374 @@
+html {
+    height: 100%;
+}
+
+body {
+    background-color: #170c1c;
+    font-family: sans-serif;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+:root {
+    --light-background-color: #211029;
+    --lighter-background-color: #281431;
+    --light-border-color: #32183e;
+    --light-border: 1px solid var(--light-border-color);
+    --muted-color: #72388c;
+    --link-color: #a04ec6;
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+    color: white;
+}
+
+header h1 {
+    margin: 0 0 0 12px;
+    font-size: 1em;
+}
+
+.title-bar {
+    background-color: var(--light-background-color);
+    border-bottom: var(--light-border);
+    padding: 16px 0;
+    font-size: 1.4em;
+}
+
+.menu-bar {
+    background-color: var(--light-background-color);
+    border-bottom: var(--light-border);
+    padding: 16px 0;
+}
+
+.menu-bar__items {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    gap: 12px;
+}
+
+.menu-bar__items a {
+    color: white;
+    padding: 0 12px;
+    text-decoration: none;
+}
+
+h2 {
+    margin: 0;
+    padding: 24px 0;
+    font-size: 1em;
+}
+
+.tasks_total_filter {
+    padding: 24px 0;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    text-align: center;
+    margin-left: 12px;
+}
+
+.tasks_total_filter span:nth-child(2) {
+    display: none;
+}
+
+.tasks_total_filter__items {
+    display: flex;
+    justify-content: center;
+    gap: 6px;
+}
+
+.tasks_total_filter__items a {
+    color: var(--link-color);
+    text-transform: capitalize;
+}
+
+.tasks__list__item.tasks__list__item--not-found {
```

Comment:
Is there some system to when you use (one or two) dashes vs. underscores?

Reply:
Just following the BEM style https://getbem.com/

Single _ being just a separator in a class name where as __ is for a child element. 

Although I seem some inconsistencies with my - or _